crates/sui-config/src/node.rs

```diff
@@ -423,8 +423,7 @@ pub struct ConsensusConfig {
 
     /// Maximum number of pending transactions to submit to consensus, including those
     /// in submission wait.
-    /// Assuming 10_000 txn tps * 10 sec consensus latency = 100_000 inflight consensus txns,
-    /// Default to 100_000.
+    /// Default to 20_000 inflight limit, assuming 20_000 txn tps * 1 sec consensus latency.
```

Comment:
1 sec consensus latency under 20K load might be a bit on the low side but I am ok with having 20K inflight txes overall.

Reply:
Sounds good. We can lower them even more in future.